Ardupilog.m

```diff
@@ -308,6 +314,53 @@ function delete(obj)
             return;
         end
 
+        function slice = getSlice(obj, slice_values, slice_type)
+        % This returns an indexed portion (a "slice") of an Ardupilog
+        % Example:
+        %    log_during_cruise = Log.getSlice([t_begin_cruise, t_end_cruise], 'TimeUS')
+        %  will return a smaller Ardupilog, only containing log data between
+        %  TimeUS values greater than t_begin_cruise and less than t_end_cruise.
+
+            % Copy all the properties, zero the number of messages
+            slice = copy(obj);
+            slice.numMsgs = 0;
+            
+            % Loop through the LogMsgGroups, slicing each one
+            for msgGroup = slice.logMsgGroups
```

Comment:
I don't understand the utility of the new logMsgGroups property. It seems to hold all the discovered message types as sub-properties, but apart from that I don't see anything else.
Especially since all the actual LogMsgGroups objects are properties of the Ardupilot object, not the Ardupilot.logMsgGroups property.